### PR TITLE
Add tests with mocha and update readme.md

### PR DESCRIPTION
**Pull request by eirikbacker**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const config = {
+  svgFileName: 'test-icons.js',
+  svgFileNameMin: 'test-icons.min.js',
+  srcPath: __dirname,
+  distPath: __dirname,
+  banner: 'Copyright'
+}
+
+// Generate test files
+before(() => svgtojs(config).then((js) => (result = js)))
```
Comment (akselnordal) [inline test/index.js]: Might be a convention, but not a requirement, bu before/after are usually wrapped inside `describe`

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+describe('svgToSymbol', () => {
```
Comment (akselnordal) [inline test/index.js]: could have been nice to add a test case that injects generated js into a dom; JSDom should be sufficient

File: test/index.js (hunk 1 above)
Comment (eirikbacker) [reply test/index.js]: Fixed :)

File: test/index.js (hunk 2 above)
Comment (eirikbacker) [reply test/index.js]: Fixed :)

**Comment by akselnordal** [comment]
Some files not included in PR should be fixed

-- .npmignore --
remove package.lock (should have a lock-in for a current published version)
*but* keep .npmignore as a file, to not inherit from .gitignore

-- .gitignore --
remove all entries
add node_modules
add editor-project files (if any, ie. webstorm config-files etc)

-- package.json --
change name to "nrk-svg-to-js" (or preferably register `@nrk` as a npm-org)
add `np` as devDependency
add scripts `publish:patch`, `publish:minor`, `publish:major` that runs `np patch`, `np minor` and `np major`

-- new file: .npmrc --
```
save=true
save-exact=true
registry=https://registry.npmjs.org/
```
(to ensure publishing is done to public npm registry)

**Pull request merged**